fix(sentry-ruby): A note on Rack apps without Rackup (#8555)
getsentry/sentry-docs: src/platform-includes/getting-started-config/ruby.rack.mdx

use Sentry::Rack::CaptureExceptions
```

### Without Rackup

<Alert level="warning">

If you're not using Rackup to run your app, then Sentry initialization code **must**
be placed before your code requires Sinatra:

</Alert>

```Ruby {filename:app.rb}
require 'sentry-ruby'

Sentry.init do |config|
  config.dsn = '___PUBLIC_DSN___'
  config.breadcrumbs_logger = [:sentry_logger, :http_logger]
end

> # Notice: requiring sinatra must go after Sentry SDK is initialized in a non-rackup environment

Reviewer: ```suggestion
# in a non-rackup environment you must initialize the Sentry SDK before requiring sinatra
```

This phrasing just sounded weird to me, so optional rephrase suggestion
